app/database/models.py

```diff
@@ -99,6 +99,7 @@ class Event(Base):
     invitees = Column(String)
     privacy = Column(String, default=PrivacyKinds.Public.name, nullable=False)
     emotion = Column(String, nullable=True)
+    image = Column(String, nullable=True)
```

Comment:
I don't think it's actually a String, unless you converted it to some textual format as base64. In any other case, it's probably Binary or something like that

Reply:
it's str as it is just the file name (similar to what was done on upload profile image)